# This message describes a robot arm movement
# Array indices from 0 to max match joints from base joint to last joint
bool tool_activation
JointConfigCommanded[] joint_moves

